jbpm-bpmn2/src/test/resources/BPMN2-BusinessRuleTaskWithDataInput.drl

```diff
@@ -21,7 +21,7 @@ import org.jbpm.bpmn2.objects.Person;
 rule "Person rule" ruleflow-group "MyRuleFlow"
 no-loop
     when
-        $p : Person()
+        $p : Person(name == null)
```

Comment:
This should not be needed. The issue seems to be with no-loop + eager evaluation directly in Drools. I believe @mariofusco fixed it for 6.5.x and master.